catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/LatLonCoordinateStrategyTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.v110.catalog.source;
+
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Envelope;
+import org.junit.Test;
+
+public class LatLonCoordinateStrategyTest {
+  private final LatLonCoordinateStrategy latLonCoordinateStrategy = new LatLonCoordinateStrategy();
+
+  @Test
+  public void toStringSingleCoordinateReturnsCommaSeparatedCoordinateValuesInLatLonOrder() {
+    assertThat(latLonCoordinateStrategy.toString(new Coordinate(10, 20)), is("20.0,10.0"));
```

Comment:
Is this type of test necessary? It seems kind of brittle

Reply:
Brittle in what way? The coordinate strings must be formatted in a certain way for geospatial queries to work.

Reply:
Brittle in that when the format changes, we would have to update all the tests that check for a certain string being returned. Not sure how often the format would change though

Reply:
We need to make sure this is formatted correctly so the query will work. It's not going to change often.